scripts/scr_star_name/scr_star_name.gml

```diff
@@ -1,595 +1,163 @@
 function scr_star_name() {
+	var rand, ok=0,num="";
+	rand=floor(random(566));
```

Comment:
instead of `566` I recommend `array_length(planet_names)`, 
also you can just have `var rand = floor(...`, simpler